web/Web/web/search/templates/search/search.html

```diff
@@ -51,124 +58,147 @@
-    <script src="{% static 'js/mousetrap.min.js' %}"></script>
-    <script src="https://unpkg.com/runtime-memcache@2.0.0"></script>
-    <script src="{% static 'js/moment.min.js' %}"></script>
-    <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
-    <script src="{% static 'js/document-viewer.js' %}"></script>
-    <script src="{% static 'js/highlight.js' %}"></script>
-
-    <script>
-
-        var context = {
-            "isQueryPage": {{ isQueryPage|yesno:"true,false"}},
-            "queryID": "{{ queryID }}",
-            "query": "{{ query }}"
-        }
-
-        var dv = new docView();
-        dv.init({
-          username: '{{ request.user }}',
-          singleDocumentMode: true,
-          judgingSourceName: "search",
-          queryID: "{{ queryID }}",
-          query: "{{ query }}",
-          searchMode: true,
-          getDocumentsToJudgeURL: '{% url 'CAL:get_docs' %}',
-          getPrevDocumentsJudgedURL: '{% url 'judgment:get_latest' number_of_docs_to_show=50 %}',
-          getDocumentIDsToJudgeURL: '{% url 'CAL:get_docs_ids' %}',
-          sendDocumentJudgmentURL: '{% url 'judgment:post_judgment' %}',
-          getDocumentURL: '{% url 'core:get_single_doc' %}?docid=',
-          fetchPreviouslyJudgedDocsOnInit: false,
-          mainJudgingCriteriaName: '{% if main_judging_criteria_name %}{{ main_judging_criteria_name }}{% else %}Relevant{% endif %}',
-          csrfmiddlewaretoken: "{{ csrf_token }}",
-
-          afterDocumentJudge: function(docid, rel) {
-            // update document indicator in SERP
-            let newColor = "#fff";
-            switch (rel) {
-              case 0:
-                newColor = "#a5a5a5"; break;
-              case 1:
-                newColor = "#c6fab8"; break;
-              case 2:
-                newColor = "#84c273"; break;
-            }
-            $(`#doc_${docid}_SERP_indicator`).attr('style', `border-color:${newColor}`);
-          },
-
-          afterDocumentLoad: function (docid) {
-            // Perform keyword search if necessary
-            searchForKeyword();
-          },
-
-          afterCALFailedToReceiveJudgment: function(docid, rel) {
-            $('#toast_CALFailedToReceiveJudgment').toast('dispose');
-            $(".CALFailedToReceiveJudgmentSpinner").removeClass("d-none");
-            $("#CALFailedToReceiveJudgment_toast_body").addClass("d-none");
-            $("#toast_CALFailedToReceiveJudgment_docid").html(docid);
-            setInterval(function(){
-              $("#CALFailedToReceiveJudgment_toast_body").removeClass("d-none");
-              $(".CALFailedToReceiveJudgmentSpinner").addClass("d-none");
-            },500);
-            $('#toast_CALFailedToReceiveJudgment').toast('show');
-          }
-        });
-
-        function HtmlDecode(s) {
-            var el = document.createElement("div");
-            el.innerHTML = s;
-            return el.innerText || el.textContent;
```

Comment:
It's much harder to track changes in the code with long unrelated changes. Let's keep it the same for now and have the logging changes added.